locales/en-US/app.l20n

```diff
@@ -104,8 +103,9 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainer "There are <span data-l10n-id="userCount" class="bold" data-hook="install-count"></span> people trying <span data-hook="title"></span> right now!">
```

Comment:
Also, because it's probably my job to make stupid comments, but do we care about the singular case?

:+1: "There are 0 people using {{title}} right now!"
:-1: "There are 1 people using {{title}} right now!" [are => is, people => person]
:+1: "There are 2 people using {{title}} right now!"

I'd almost lean towards "meh, f-it", because it's a pretty small and blunt edge and not worth the added complexity, and words are hard.

---

**NOTE:** If it **_isn't_** my job to make stupid comments, I'd like to apologize for the past 2.5 years of stupidity and GIFs.